production-marathon/marathon.json.b

```diff
@@ -1,43 +1,40 @@
 {
-	"id": "caf",
-	"groups": [{
-		"id": "audit",
-		"apps": [{
-			"id": "audit-service",
-			"cpus": 0.25,
-			"mem": 768,
-			"instances": 1,
-			"container": {
-				"docker": {
-					"image": "${DOCKER_REGISTRY}/audit-service:3.1.0",
-					"network": "BRIDGE",
-					"portMappings": [{
-						"containerPort": 8080,
-						"hostPort": 0,
-						"protocol": "tcp",
-						"servicePort": ${CAF_AUDIT_SERVICE_PORT}
-					}],
-					"forcePullImage": true
-				},
-				"type": "DOCKER"
-			},
-			"env": {
-				"_JAVA_OPTIONS": "-Xms384m -Xmx384m",
-				"CAF_ELASTIC_HOST_AND_PORT": "${CAF_ELASTIC_HOST_AND_PORT}",
-				"CAF_ELASTIC_CLUSTER_NAME": "${CAF_ELASTIC_CLUSTER_NAME}",
-				"CAF_ELASTIC_NUMBER_OF_REPLICAS": "1",
-				"CAF_ELASTIC_NUMBER_OF_SHARDS": "5",
-				"CAF_LOG_LEVEL": "INFO"
-			},
-			"healthChecks": [{
-				"gracePeriodSeconds": 300,
-				"intervalSeconds": 60,
-				"maxConsecutiveFailures": 3,
-				"path": "/",
-				"portIndex": 0,
-				"protocol": "HTTP",
-				"timeoutSeconds": 20
-			}]
-		}]
-	}]
+    "id": "audit-prod",
```

Comment:
The root level caf group has been removed because it is defined in the curl command now. You cannot add apps to existing groups so apps that each have their own JSON essentially need to contained in their own group. The name of the group was adjusted to audit-prod under Dermot's instruction. In contrast, when ES is deployed it is deployed under ../audit-testing/ .

It is not likely to be consistent with JobService; I found this issue yesterday so it is more than likely that JobService will need the same changes applied.